web/src/main/kotlin/no/nav/su/se/bakover/web/features/SuUserFeature.kt

```diff
@@ -52,12 +62,27 @@ class SuUserFeature(configuration: Configuration) {
 
         val authHeader = context.call.request.header(HttpHeaders.Authorization)
 
-        suUserContext.setUser(
-            if (authHeader != null) {
-                clients.microsoftGraphApiClient.hentBrukerinformasjon(authHeader)
-                    .mapLeft { KallMotMicrosoftGraphApiFeilet(it) }
-            } else {
-                Either.Left(ManglerAuthHeader)
+        suUserContext.init(
+            user =
+                if (authHeader != null) {
+                    clients.microsoftGraphApiClient.hentBrukerinformasjon(authHeader)
+                        .mapLeft { KallMotMicrosoftGraphApiFeilet(it) }
+                } else {
+                    Either.Left(ManglerAuthHeader)
+                },
+            asserter = object : PersontilgangAsserter {
+                override fun assertBrukerHarTilgangTilPerson(fnr: Fnr): Person =
+                    clients.personOppslag.person(fnr)
+                        .getOrHandle { throw Tilgangssjekkfeil(it, fnr) }
+
+                override fun assertBrukerHarTilgangTilPerson(sakId: UUID): Either<FantIkkeSak, Pair<Person, Sak>> =
+                    services.sak.hentSak(sakId)
+                        .map { sak ->
+                            context.call.audit("Hentet sak med id: $sakId")
```

Comment:
Henter sak for  å sjekke om bruker har tilgang til sak eller no

Reply:
Tenker du på loggmeldinga?
Føler den er litt upresis, for den hentes ikke her _kun_ for å sjekke tilgang, men også for å bruke den videre.